src/commands/mod.rs

```diff
@@ -18,8 +17,9 @@ pub mod summon;
 pub mod version;
 
 pub enum PlayFlag {
-    DEFAULT,
-    PLAYTOP,
+    END,
+    NEXT,
+    ALL,
```

Comment:
While we are at it, refactor this enum to be something like `PlayMode` what do you think @aquelemiguel ? Also, usually enum values are PascalCase (so All, End, Next).

Reply:
Yeah, I like it, let's drop the flag naming.

Reply:
Circling back to this conversation due to the PascalCase enums. In Rust, Enums are in Pascal case, which we can already find in our code in the `Connection` enum. However, it is true that the old `EnqueueType` and `PlayFlag` were all capitalised... Either way, we should move to the standard imo.